nixos/modules/config/networking.nix

```diff
@@ -190,7 +251,10 @@ in
 
     environment.etc =
       { # /etc/services: TCP/UDP port assignments.
-        services.source = pkgs.iana-etc + "/etc/services";
+        services.source =
+          if cfg.services == {}
+          then "${pkgs.iana-etc}/etc/services"
```

Comment:
There should really be a package option for iana-etc because it triggeres a rebuild for go and a bunch of python things.

Reply:
Well its just `iana-etc` and only `iana-etc` right now. If people want a package option I think that can be another PR.